deploy/stacks/cloudfront.py

```diff
@@ -42,10 +42,15 @@ def __init__(
     ):
         super().__init__(scope, id, **kwargs)
 
```

Comment:
With this change we are creating a dedicated bucket for the access logs in the FE region right?

Reply:
that's correct, in case they are in the same region we create a second bucket nevertheless